yarn-error.log

```diff
@@ -0,0 +1,5996 @@
+Arguments: 
```

Comment:
yarn-error.log should be gitignored.

Reply:
Added to .gitignore

Reply:
Great! Note that the file is still tracked currently.